config/environments/production.rb

```diff
@@ -96,10 +96,10 @@
   config.action_mailer.smtp_settings = {
     :port                 => ENV['SMTP_PORT'],
     :address              => ENV['SMTP_SERVER'],
-    :user_name            => ENV['SMTP_LOGIN'],
-    :password             => ENV['SMTP_PASSWORD'],
+    :user_name            => ENV['SMTP_LOGIN'] == 'none' ? nil : ENV['SMTP_LOGIN'].presence,
+    :password             => ENV['SMTP_LOGIN'] == 'none' ? nil : ENV['SMTP_PASSWORD'].presence,
```

Comment:
pretty sure this should say `ENV['SMTP_PASSWORD'] == 'none' ? nil : ENV['SMTP_PASSWORD'].presence,`